part7/anecdotes/src/components/Footer.js

```diff
@@ -1,11 +1,43 @@
 import React from 'react'
 
+import styled from '@emotion/styled'
+import css from '@styled-system/css'
+
+import Space from '../design/space'
+import TextBlock from '../design/textblock'
```

Comment:
Komponenšu vārdus un failu nosaukumus parasti definējam PascalCase notācijā.

Ref: https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841